javafx/javafx2.samples/src/org/netbeans/modules/javafx2/samples/resources/layer.xml

```diff
@@ -24,351 +24,353 @@
-                    </file>
-
-                    <file name="TransitionPath.zip" url="TransitionPath.zip">
-                        <attr name="displayName" bundlevalue="org.netbeans.modules.javafx2.samples.Bundle#Templates/Project/Samples/JavaFX/TransitionPath.zip"/>
-                        <attr name="iconBase" stringvalue="org/netbeans/modules/javafx2/project/ui/resources/jfxjava.png"/>
-                        <attr name="position" intvalue="3200"/>
-                        <attr name="template" boolvalue="true"/>
-                        <attr name="instantiatingIterator" newvalue="org.netbeans.modules.javafx2.samples.JavaFXSampleProjectIterator"/>
-                        <attr name="instantiatingWizardURL" urlvalue="nbresloc:/org/netbeans/modules/javafx2/samples/resources/TransitionPath.html"/>
-                        <attr name="prjType" stringvalue="javafx2"/>
-                    </file>
-
-                    <file name="TransitionRotate.zip" url="TransitionRotate.zip">
-                        <attr name="displayName" bundlevalue="org.netbeans.modules.javafx2.samples.Bundle#Templates/Project/Samples/JavaFX/TransitionRotate.zip"/>
-                        <attr name="iconBase" stringvalue="org/netbeans/modules/javafx2/project/ui/resources/jfxjava.png"/>
-                        <attr name="position" intvalue="3300"/>
-                        <attr name="template" boolvalue="true"/>
-                        <attr name="instantiatingIterator" newvalue="org.netbeans.modules.javafx2.samples.JavaFXSampleProjectIterator"/>
-                        <attr name="instantiatingWizardURL" urlvalue="nbresloc:/org/netbeans/modules/javafx2/samples/resources/TransitionRotate.html"/>
-                        <attr name="prjType" stringvalue="javafx2"/>
-                    </file>
-
-                    <file name="WebViewBrowser.zip" url="WebViewBrowser.zip">
-                        <attr name="displayName" bundlevalue="org.netbeans.modules.javafx2.samples.Bundle#Templates/Project/Samples/JavaFX/WebViewBrowser.zip"/>
-                        <attr name="iconBase" stringvalue="org/netbeans/modules/javafx2/project/ui/resources/jfxjava.png"/>
-                        <attr name="position" intvalue="3400"/>
-                        <attr name="template" boolvalue="true"/>
-                        <attr name="instantiatingIterator" newvalue="org.netbeans.modules.javafx2.samples.JavaFXSampleProjectIterator"/>
-                        <attr name="instantiatingWizardURL" urlvalue="nbresloc:/org/netbeans/modules/javafx2/samples/resources/WebViewBrowser.html"/>
-                        <attr name="prjType" stringvalue="javafx2"/>
-                    </file>
-
+                <folder name="Standard">
+                    <folder name="JavaFX">
+                        <attr name="position" intvalue="110"/>
+                        <attr name="displayName" bundlevalue="org.netbeans.modules.javafx2.samples.Bundle#Templates/Project/Samples/JavaFX"/>
```

Comment:
This bundle value does not exist anymore. See the change in Bundle.properties, line 25: https://github.com/apache/netbeans/blob/df832297fc4fac4d6f3549b8202564e56958f4e4/javafx/javafx2.samples/src/org/netbeans/modules/javafx2/samples/Bundle.properties#L25